src/java.base/share/classes/java/io/ObjectOutputStream.java

```diff
@@ -585,7 +585,7 @@ protected void annotateProxyClass(Class<?> cl) throws IOException {
      * substituted or the original object.
      *
      * <p>Null can be returned as the object to be substituted, but may cause
-     * NullReferenceException in classes that contain references to the
+     * {@code NullPointerException} in classes that contain references to the
```

Comment:
Maybe use `{@link …}` instead of `{@code …}`?
```suggestion
     * {@link NullPointerException} in classes that contain references to the
```

Reply:
In this case I don't have a specific preference. I've updated the PR to use `@link` instead of `@code`.